cryostatio/cryostat-web
# feat(graphql): recording metadata/labels handling

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Related to: [315](https://github.com/cryostatio/cryostat3/pull/315)
Depends on: [319](https://github.com/cryostatio/cryostat3/pull/319)
## Description of the change:
*This change adds allows a match expression example to be copied to the clipboard...*

## Motivation for the change:
*This change is helpful because users may want to copy the example for easier use...*

## How to manually test:
1. see [https://github.com/cryostatio/cryostat3/pull/319](https://github.com/cryostatio/cryostat3/pull/319)
